src/cargo_ops/temp_project.rs

```diff
@@ -40,7 +40,7 @@ impl<'tmp> TempProject<'tmp> {
         let workspace_root = orig_workspace.workspace.root();
         let workspace_root_str = workspace_root.to_string_lossy();
 
-        let temp_dir = TempDir::new("cargo-outdated")?;
+        let temp_dir = Builder::new().prefix("cargo-outdated").tempdir()?;
```

Comment:
If it's not necessary for the temporary directory name to start with `cargo-outdated`, this line could be changed to just

```
let temp_dir = TempDir::new()?;
```